docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,330 @@
-and run once the required bundled libraries (NCEPLIBS) and third-party libraries (NCEPLIBS-external)
-are built.
-
-Limited-Test (level 3) and Build-Only (level 4) computational platforms are those in which
-the developers have built the code but little or no pre-release testing has been conducted,
-respectively. A complete description of the levels of support, along with a list of preconfigured
-and configurable platforms can be found in the `SRW Application wiki page 
-<https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_.
-
-User Support, Documentation, and Contributing Development
-=========================================================
-
-A forum-based, online `support system <https://forums.ufscommunity.org>`_ with topical sections
-provides a centralized location for UFS users and developers to post questions and exchange
-information. The forum complements the formal, written documentation, summarized here for ease of
-use.  
+----------------------------
+
+The SRW Application has a portable CMake-based build system that packages together all the components required to build the SRW Application. Once built, users can generate a Rocoto-based workflow that will run each task in the proper sequence (see `Rocoto documentation <https://github.com/christopherwharrop/rocoto/wiki/Documentation>`__ for more on workflow management). Individual components can also be run in a stand-alone, command line fashion. 
+
+The SRW Application allows for configuration of various elements of the workflow. For example, users can modify the parameters of the atmospheric model, such as start and end dates, duration, time step, and the physics suite for the forecast. 
+
+This SRW Application release has been tested on a variety of platforms widely used by researchers, including NOAA High-Performance Computing (HPC) systems (e.g. Hera, Orion), cloud environments, and generic Linux and macOS systems. Four `levels of support <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ have been defined for the SRW Application. Preconfigured (Level 1) systems already have the required external libraries (e.g., NCEPLIBS) available in a central location. The SRW Application is expected to build and run out-of-the-box on these systems, and users can :ref:`download the SRW code <DownloadSRWApp>` without first installing prerequisites. On other platforms, the SRW can be :ref:`run within a container <QuickstartC>` that includes the HPC-Stack, or the required libraries will need to be installed as part of the :ref:`SRW build <BuildRunSRW>` process. Once these prerequisite libraries are installed, applications and models should build and run successfully. However, users may need to perform additional troubleshooting on Level 3 or 4 systems since little or no pre-release testing has been conducted on these systems. 
```

Comment:
Should we specify in the first sentence that this is release v2?  So, "The SRW App v2.0.0" or whatever it will be?

Reply:
We should probably stick with the numbering convention that was used with the first release (which was based on the first MRW App release).  The first release was v1.0.0, followed by a bug-fix v1.0.1 release, so logically, this would be v2.0.0.

Reply:
Sounds good to me! The one concern is that if I update the version number on the docs now, people might be looking for the v2.0.0 tag, which won't exist for several weeks yet. Shall I do it anyway?

Reply:
Yeah, let's hold off on that.  Ideally, this documentation should be general enough for either the release or develop branch code.  We can refer to specific releases, but since v2.0.0 doesn't exist yet, we can save that for another PR.

Reply:
Right. Technically that should be done in the release branch once it is cut. The trick is remembering and catching all those!